src/main/java/tuteez/logic/commands/DeleteCommand.java

```diff
@@ -61,6 +61,15 @@ public CommandResult execute(Model model) throws CommandException {
             personToDelete = getPersonToDeleteByName(model, targetName);
         }
 
+        if (model.getLastViewedPerson().get().isPresent()) {
+            if (personToDelete.equals(model.getLastViewedPerson().get().get())) {
```

Comment:
I believe it is better to not chain so many getters. Maybe you can abstract the method `model.getLastViewedPerson().get().isPresent()` and `model.getLastViewedPerson().get().get())`

Reply:
Noted, will do!